tests/zenko_tests/node_tests/cloudserver/bucketPolicy/tests/bucketPolicy.js

```diff
@@ -0,0 +1,565 @@
+            Prefix: '',
+            Expiration: {
+                Days: 1,
+            },
+        },
+    ],
+};
+const repConfig = {
+    Role: 'arn:aws:iam::account-id:role/src-resource',
+    Rules: [
+        {
+            Destination: {
+                Bucket: `arn:aws:s3:::${bucket}`,
+                StorageClass: 'STANDARD',
+            },
+            Prefix: 'test-prefix',
+            Status: 'Enabled',
+            ID: 'test-id',
+        },
+    ],
+};
+const vConfig = {
+    Status: 'Enabled',
+};
+
+const basicStatement = {
+    Sid: 'statementid',
+    Effect: 'Allow',
+    Principal: '*',
+    Action: ['s3:GetBucketTagging'],
+    Resource: [
+        `arn:aws:s3:::${bucket}`,
+    ],
+};
+
+const objArn = `arn:aws:s3:::${bucket}/*`;
+const allow = 'allow alt account to perform specified action';
+
+function getPolicyParams(paramsToChange) {
+    const newParam = {};
+    const bucketPolicy = {
+        Version: '2012-10-17',
+        Statement: [basicStatement],
+    };
+    if (paramsToChange) {
+        for (let i = 0; i < paramsToChange.length; i++) {
+            newParam[paramsToChange[i].key] = paramsToChange[i].value;
+            bucketPolicy.Statement[0] = Object.assign(
+                {}, basicStatement, newParam);
+        }
+    }
+    return {
+        Bucket: bucket,
+        Policy: JSON.stringify(bucketPolicy),
+    };
+}
+
+describe('Bucket policies', () => {
+    beforeEach(done => {
+        scalityS3Client.createBucket({
+            Bucket: bucket,
+            CreateBucketConfiguration: { LocationConstraint: 'us-east-1' } },
+        done);
+    });
+
+    afterEach(done => testUtils.emptyDeleteBucket(bucket, done));
+
+    describe('with no bucket policy', () => {
+        it('should deny request from alternate account', done => {
+            altScalityS3Client.getBucketLocation(bParam, err => {
+                assert.strictEqual(err.code, 'AccessDenied');
+                done();
+            });
+        });
+    });
+
+    describe('with basic bucket policy', () => {
+        it(`should ${allow}: Abort MPU`, done => {
+            const params = getPolicyParams([
+                { key: 'Action', value: 's3:AbortMultipartUpload' },
+                { key: 'Resource', value: objArn },
+            ]);
+            async.waterfall([
+                next => scalityS3Client.putBucketPolicy(params, next),
+                (policyData, next) => scalityS3Client.createMultipartUpload(
+                    { Bucket: bucket, Key: mpuKey }, next),
+                (mpuData, next) => {
+                    const uId = mpuData.UploadId;
+                    altScalityS3Client.abortMultipartUpload(
+                        { Bucket: bucket, Key: mpuKey, UploadId: uId }, next);
+                },
+            ], err => {
+                assert.ifError(err);
+                done();
+            });
+        });
+
+        // DeleteBucket test either causes 'bucket not found' error on cleanup
```

Comment:
Could you achieve this by moving this in its own `describe` block?

Reply:
oh yeah 🤦‍♀